unifi-linuxserver.json

```
@@ -83,13 +83,13 @@
             }
         },
         "description": "Unifi Access Point controller",
-        "more_info": "This is a containerized version of Ubiquiti Networks Unifi Controller.",
+        "more_info": "This is a containerized version of Ubiquiti Networks Unifi Controller. To limit memory add MEM_LIMIT=1024M as a custom argument after install",
         "volume_add_support": false,
         "ui": {
             "https": true,
             "slug": ""
         },
-        "website": "https://hub.docker.com/r/linuxserver/unifi/",
-        "version": "1.0"
+        "website": "https://hub.docker.com/r/linuxserver/unifi-controller/",
```

Link to upstream project:
```suggestion
        "website": "https://www.ubnt.com/enterprise/#unifi",
```